src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -12,12 +13,22 @@ import { useServerData } from '../../../lib/providers';
 import { currencyFormat } from '../../../lib/hooks/currency'; // Remove .00 cents from formated min and max
 
 export const ShortTerm = ({
-    content: { instructions, linkToProductList, estimatedInstallments, disclosure, donutTimestamps, learnMoreLink },
-    productMeta: { qualifying, periodicPayment },
+    content: {
+        instructions,
+        linkToProductList,
+        estimatedInstallments,
+        preapproval,
+        disclosure,
+        donutTimestamps,
+        learnMoreLink
```

Comment:
```suggestion
        learnMoreLink,
        cta
```
Let's import the `cta` content into the component so we can use it in our button.